docs/intro/reusable-apps.txt

```diff
@@ -181,50 +181,48 @@ this. For a small app like polls, this process isn't too difficult.
    license. Just be aware that your licensing choice will affect who is able
    to use your code.
 
-#. Next we'll create a ``setup.py`` file which provides details about how to
-   build and install the app.  A full explanation of this file is beyond the
-   scope of this tutorial, but the `setuptools docs
-   <https://setuptools.readthedocs.io/en/latest/>`_ have a good
-   explanation. Create a file ``django-polls/setup.py`` with the following
-   contents:
+#. Next we'll create ``setup.cfg`` and ``setup.py`` files which provide details
+   about how to build and install the app. A full explanation of these files is
+   beyond the scope of this tutorial, but the `setuptools docs
+   <https://setuptools.readthedocs.io/en/latest/>`_ have a good explanation.
+   Create the files ``django-polls/setup.cfg`` and ``django-polls/setup.py``
+   with the following contents:
+
+   .. code-block:: ini
+        :caption: django-polls/setup.cfg
+
+        [metadata]
+        name = django-polls
+        version = 0.1
+        description = A Django app to conduct Web-based polls.
+        long_description = file: README.rst
+        url = https://www.example.com/
+        author = Your Name
+        author_email = yourname@example.com
+        license = BSD License  # example license
+        classifiers =
+            Environment :: Web Environment
+            Framework :: Django
+            Framework :: Django :: X.Y  # replace "X.Y" as appropriate
+            Intended Audience :: Developers
+            License :: OSI Approved :: BSD License  # example license
+            Operating System :: OS Independent
+            Programming Language :: Python
+            Programming Language :: Python :: 3.6
+            Programming Language :: Python :: 3.7
```

Comment:
Please also add the classifier for Python 3.8 to match supported versions for 3.1.

(Care should be taken to remove it again if these changes are backported.)

Reply:
We should probably also add `Programming Language :: Python :: 3 :: Only`.